.github/ISSUE_TEMPLATE/bug_ivv_documentation.md

```diff
@@ -0,0 +1,53 @@
+---
+name: 🐞 Bug IVV
```

Comment:
Please add documentation (or DOC) in the name :)

Reply:
DOC added